.github/workflows/gsbuild.yml

```diff
@@ -0,0 +1,43 @@
+name: GS Test Build
+
+on:
+  pull_request:
+
+jobs:
+  build-roman:
+    runs-on: ubuntu-latest
+    name: Build Roman variable font
+    steps:
+      - name: Check out font project source repository
+        uses: actions/checkout@v4
+        with:
+          repository: googlefonts/googlesans
+          token: ${{ secrets.GS_READ_FONTC }}
+      - name: Install the latest stable Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: Install fontc and compile fonts
+        uses: googlefonts/fontc-action@v1
+        with:
+          source-path: source/GoogleSans/GoogleSans.designspace
+          debug: true
+
+  build-italic:
+    runs-on: ubuntu-latest
+    name: Build Italic variable font
```

Comment:
If compiling the compiler is by far the slowest step  it's possible that building Romand and Italic in one job would make sense?

Reply:
7db4cb1f058d0a01e687312099dd784390dd904f

Reply:
With the current approach the fontc compile takes about 50s and each variable font compile takes 1.5 mins.  OK to keep this sequential approach or should we run the builds in parallel again?